pkg/descheduler/strategies/nodeutilization/lownodeutilization_test.go

```diff
@@ -643,6 +643,112 @@ func TestLowNodeUtilization(t *testing.T) {
 			},
 			expectedPodsEvicted: 3,
 		},
+		{
+			name: "without priorities, but only other node doesn't match pod node affinity for p4 and p5",
```

Comment:
I don't see any pod node affinities set in this test. Would you elaborate on what this test is expected to test?